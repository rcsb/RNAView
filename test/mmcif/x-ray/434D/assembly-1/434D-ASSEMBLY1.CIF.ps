%!PS-Adobe-2.0
%%Title: A postscript generated from a RNAML file.
%%CreationDate: Wed Nov 15 11:12:37 2023
%%Orientation: Portrait
%%BoundingBox:    251   289   361   504

/Times-Bold findfont 12  scalefont setfont


/Times-Bold findfont 13  scalefont setfont    %%set font for (A U G C T)

%% stacking diagram widths for base-pairs 1 & 2
/W1 {1 setlinewidth} bind def
/W2 {1.5 setlinewidth} bind def
/W3 {2 setlinewidth} bind def
/W4 {3 setlinewidth} bind def

%% minor and major grooves filling color saturation
/MINOR_SAT 0.9 def
/MAJOR_SAT 0.1 def
/OTHER_SIDES {0 0 1 sethsbcolor fill} bind def

%% defineing geometry shapes
/NP {newpath} bind def
/CIRCLE {0 360 arc closepath} bind def                        %% circle 
/TRIANGLE {moveto lineto lineto closepath} bind def           %% triangle
/SQUARE {moveto lineto lineto lineto closepath} bind def      %% square
/LINE {moveto lineto stroke} bind def                         %% line
/DASHLINE { moveto lineto [2 4] 0 setdash stroke } bind def %% dashline
/DOTLINE  {1 0 1 setrgbcolor} bind def                        %% dotline
/Dw {1 setlinewidth} bind def
/FB {setgray fill} bind def
/R6 {moveto lineto lineto lineto lineto lineto closepath} bind def
/R9 {moveto lineto lineto lineto lineto lineto
     lineto lineto lineto closepath} bind def

%% line drawing colors for ACGITU & others
/Al {0.0000 1.00 1.00 sethsbcolor} bind def
/Cl {0.1667 1.00 1.00 sethsbcolor} bind def
/Gl {0.3333 1.00 1.00 sethsbcolor} bind def
/Il {0.3333 1.00 0.57 sethsbcolor} bind def
/Tl {0.6667 1.00 1.00 sethsbcolor} bind def
/Ul {0.5000 1.00 1.00 sethsbcolor} bind def
/Xl {0.0000 0.00 0.00 sethsbcolor} bind def
/XX {0.2000 0.20 0.20 sethsbcolor} bind def
/FILLBLACK {0.0000 0.00 0.00 sethsbcolor fill} bind def

%% minor groove filling colors for ACGITU & others
/Am {0.0000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Cm {0.1667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Gm {0.3333 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Im {0.3333 MINOR_SAT 0.57      sethsbcolor fill} bind def
/Tm {0.6667 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Um {0.5000 MINOR_SAT 1.00      sethsbcolor fill} bind def
/Xm {0.0000 0.00      MAJOR_SAT sethsbcolor fill} bind def

%% major groove filling colors for ACGITU & others
/AM {0.0000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/CM {0.1667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/GM {0.3333 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/IM {0.3333 MAJOR_SAT 0.57      sethsbcolor fill} bind def
/TM {0.6667 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/UM {0.5000 MAJOR_SAT 1.00      sethsbcolor fill} bind def
/XM {0.0000 0.00      MINOR_SAT sethsbcolor fill} bind def

%% define line width, line join style & cap style (1 means round)
1 setlinewidth 1 setlinejoin 1 setlinecap
   271   309 translate

/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
0.00 171.00  moveto (U) center show 
0.00 141.83  moveto (A) center show 
0.00 112.67  moveto (G) center show 
0.00 83.50  moveto (C) center show 
0.00 54.33  moveto (U) center show 
0.00 25.17  moveto (C) center show 
0.00 -4.00  moveto (C) center show 
70.00 -4.00  moveto (G) center show 
70.00 25.17  moveto (G) center show 
70.00 54.33  moveto (G) center show 
70.00 83.50  moveto (G) center show 
70.00 112.67  moveto (C) center show 
70.00 141.83  moveto (U) center show 
70.00 171.00  moveto (A) center show 
0.005 setlinewidth 1 setlinejoin 1 setlinecap
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al -0.00 181.00 moveto (5') center show grestore
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 0.00 -14.00 moveto (3') center show grestore
Il 0.00 175.00 0.00 145.83 LINE
 Il 0.00 145.83 0.00 116.67 LINE
 Il 0.00 116.67 0.00 87.50 LINE
 Il 0.00 87.50 0.00 58.33 LINE
 Il 0.00 58.33 0.00 29.17 LINE
 Il 0.00 29.17 0.00 0.00 LINE
 /Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 -14.00 moveto (5') center show grestore
/Times findfont 11 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Al 70.00 181.00 moveto (3') center show grestore
Tl 70.00 0.00 70.00 29.17 LINE
 Tl 70.00 29.17 70.00 58.33 LINE
 Tl 70.00 58.33 70.00 87.50 LINE
 Tl 70.00 87.50 70.00 116.67 LINE
 Tl 70.00 116.67 70.00 145.83 LINE
 Tl 70.00 145.83 70.00 175.00 LINE
 /Times findfont 8 scalefont setfont
/center{dup stringwidth pop 2 div neg 0 rmoveto } bind def
gsave Tl -9.00 54.33 moveto (5) center show grestore
gsave Tl 79.00 54.33 moveto (10) center show grestore
NP W1
 Xl 8.00 175.00 62.00 175.00 LINE
 NP W1
 Xl 8.00 145.83 62.00 145.83 LINE
 NP W1
 Xl 8.00 114.67 62.00 114.67 LINE
 Xl 8.00 118.67 62.00 118.67 LINE
 NP W1
 Xl 8.00 85.50 62.00 85.50 LINE
 Xl 8.00 89.50 62.00 89.50 LINE
 NP W1
 Xl 35.00 58.33 3.00 CIRCLE 
 gsave  grestore stroke
 NP W1
 Xl 8.00 27.17 62.00 27.17 LINE
 Xl 8.00 31.17 62.00 31.17 LINE
 NP W1
 Xl 8.00 -2.00 62.00 -2.00 LINE
 Xl 8.00 2.00 62.00 2.00 LINE
 
 showpage 
