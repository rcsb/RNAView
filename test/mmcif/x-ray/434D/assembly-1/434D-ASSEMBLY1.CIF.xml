<?xml version="1.0"?>
<!DOCTYPE rnaml SYSTEM "rnaml.dtd">

<rnaml version="1.0">

   <molecule id="1">
      <sequence>
         <numbering-system id="1" used-in-file="false">
            <numbering-range>
               <start>1</start>
               <end>7</end>
            </numbering-range>
         </numbering-system>
         <numbering-table length="7" comment="sequence number in pdb file">
            1    2    3    4    5    6    7 
         </numbering-table>
         <seq-data>
            UAGCUCC
         </seq-data>
      </sequence>
      <structure>
         <model id="?">
            <model-info>
               <method>Crystallography ?</method>
               <resolution>? Angstroms</resolution>
            </model-info>
            <base>
               <position>1</position>
               <base-type>U</base-type>
               <atom serial="6">
                  <atom-type> O3'</atom-type>
                     <coordinates>-10.332 -5.955 -1.818</coordinates>
               </atom>
            </base>
            <base>
               <position>2</position>
               <base-type>A</base-type>
               <atom serial="18">
                  <atom-type> P  </atom-type>
                     <coordinates>-9.097 -5.518 -2.711</coordinates>
               </atom>
               <atom serial="26">
                  <atom-type> O3'</atom-type>
                     <coordinates>-9.045 -1.326 -5.731</coordinates>
               </atom>
            </base>
            <base>
               <position>3</position>
               <base-type>G</base-type>
               <atom serial="40">
                  <atom-type> P  </atom-type>
                     <coordinates>-7.622 -1.089 -6.385</coordinates>
               </atom>
               <atom serial="48">
                  <atom-type> O3'</atom-type>
                     <coordinates>-5.680 3.168 -8.017</coordinates>
               </atom>
            </base>
            <base>
               <position>4</position>
               <base-type>C</base-type>
               <atom serial="63">
                  <atom-type> P  </atom-type>
                     <coordinates>-4.241 2.784 -8.582</coordinates>
               </atom>
               <atom serial="71">
                  <atom-type> O3'</atom-type>
                     <coordinates>0.064 5.428 -7.926</coordinates>
               </atom>
            </base>
            <base>
               <position>5</position>
               <base-type>U</base-type>
               <atom serial="83">
                  <atom-type> P  </atom-type>
                     <coordinates>1.079 4.471 -8.653</coordinates>
               </atom>
               <atom serial="91">
                  <atom-type> O3'</atom-type>
                     <coordinates>5.993 4.760 -6.523</coordinates>
               </atom>
            </base>
            <base>
               <position>6</position>
               <base-type>C</base-type>
               <atom serial="103">
                  <atom-type> P  </atom-type>
                     <coordinates>7.007 3.746 -7.179</coordinates>
               </atom>
               <atom serial="111">
                  <atom-type> O3'</atom-type>
                     <coordinates>10.933 1.559 -4.617</coordinates>
               </atom>
            </base>
            <base>
               <position>7</position>
               <base-type>C</base-type>
               <atom serial="123">
                  <atom-type> P  </atom-type>
                     <coordinates>11.574 0.370 -5.451</coordinates>
               </atom>
               <atom serial="131">
                  <atom-type> O3'</atom-type>
                     <coordinates>13.685 -3.842 -3.499</coordinates>
               </atom>
            </base>
            <str-annotation>
            </str-annotation>
            <secondary-structure-display comment="x,y coodinates">
               <ss-base-coord>
                  <base-id><position>1</position></base-id>
                  <coordinates>0.000 0.000</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>2</position></base-id>
                  <coordinates>0.000 91.667</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>3</position></base-id>
                  <coordinates>0.000 183.333</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>4</position></base-id>
                  <coordinates>0.000 275.000</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>5</position></base-id>
                  <coordinates>0.000 366.667</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>6</position></base-id>
                  <coordinates>0.000 458.333</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>7</position></base-id>
                  <coordinates>0.000 550.000</coordinates>
               </ss-base-coord>
            </secondary-structure-display>
         </model>
      </structure>
   </molecule>

   <molecule id="2">
      <sequence>
         <numbering-system id="2" used-in-file="false">
            <numbering-range>
               <start>1</start>
               <end>7</end>
            </numbering-range>
         </numbering-system>
         <numbering-table length="7" comment="sequence number in pdb file">
            8    9   10 
           11   12   13   14 
         </numbering-table>
         <seq-data>
            GGG GCUA
         </seq-data>
      </sequence>
      <structure>
         <model id="?">
            <model-info>
               <method>Crystallography ?</method>
               <resolution>? Angstroms</resolution>
            </model-info>
            <base>
               <position>1</position>
               <base-type>G</base-type>
               <atom serial="148">
                  <atom-type> O3'</atom-type>
                     <coordinates>1.237 -11.439 1.485</coordinates>
               </atom>
            </base>
            <base>
               <position>2</position>
               <base-type>G</base-type>
               <atom serial="163">
                  <atom-type> P  </atom-type>
                     <coordinates>0.719 -10.176 2.268</coordinates>
               </atom>
               <atom serial="171">
                  <atom-type> O3'</atom-type>
                     <coordinates>3.570 -7.343 5.302</coordinates>
               </atom>
            </base>
            <base>
               <position>3</position>
               <base-type>G</base-type>
               <atom serial="186">
                  <atom-type> P  </atom-type>
                     <coordinates>2.741 -6.099 5.827</coordinates>
               </atom>
               <atom serial="194">
                  <atom-type> O3'</atom-type>
                     <coordinates>4.789 -1.691 7.282</coordinates>
               </atom>
            </base>
            <base>
               <position>4</position>
               <base-type>G</base-type>
               <atom serial="209">
                  <atom-type> P  </atom-type>
                     <coordinates>3.476 -0.886 7.699</coordinates>
               </atom>
               <atom serial="217">
                  <atom-type> O3'</atom-type>
                     <coordinates>3.115 4.034 7.240</coordinates>
               </atom>
            </base>
            <base>
               <position>5</position>
               <base-type>C</base-type>
               <atom serial="232">
                  <atom-type> P  </atom-type>
                     <coordinates>1.646 4.362 7.800</coordinates>
               </atom>
               <atom serial="240">
                  <atom-type> O3'</atom-type>
                     <coordinates>-1.187 8.472 6.120</coordinates>
               </atom>
            </base>
            <base>
               <position>6</position>
               <base-type>U</base-type>
               <atom serial="252">
                  <atom-type> P  </atom-type>
                     <coordinates>-2.578 8.276 6.877</coordinates>
               </atom>
               <atom serial="260">
                  <atom-type> O3'</atom-type>
                     <coordinates>-6.921 9.846 4.306</coordinates>
               </atom>
            </base>
            <base>
               <position>7</position>
               <base-type>A</base-type>
               <atom serial="272">
                  <atom-type> P  </atom-type>
                     <coordinates>-8.254 9.535 5.149</coordinates>
               </atom>
               <atom serial="280">
                  <atom-type> O3'</atom-type>
                     <coordinates>-12.861 8.668 3.057</coordinates>
               </atom>
            </base>
            <str-annotation>
            </str-annotation>
            <secondary-structure-display comment="x,y coodinates">
               <ss-base-coord>
                  <base-id><position>1</position></base-id>
                  <coordinates>220.000 550.000</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>2</position></base-id>
                  <coordinates>220.000 458.333</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>3</position></base-id>
                  <coordinates>220.000 366.667</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>4</position></base-id>
                  <coordinates>220.000 275.000</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>5</position></base-id>
                  <coordinates>220.000 183.333</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>6</position></base-id>
                  <coordinates>220.000 91.667</coordinates>
               </ss-base-coord>
               <ss-base-coord>
                  <base-id><position>7</position></base-id>
                  <coordinates>220.000 0.000</coordinates>
               </ss-base-coord>
            </secondary-structure-display>
         </model>
      </structure>
   </molecule>


   <interactions>
            <str-annotation>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>1</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>7</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>-</edge-5p>
                  <edge-3p>-</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>2</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>6</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>-</edge-5p>
                  <edge-3p>-</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>3</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>5</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>+</edge-5p>
                  <edge-3p>+</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>4</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>4</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>+</edge-5p>
                  <edge-3p>+</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>5</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>3</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>W</edge-5p>
                  <edge-3p>W</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>6</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>2</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>+</edge-5p>
                  <edge-3p>+</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <base-pair comment="?">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>7</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>1</position>
                     </base-id>
                  </base-id-3p>
                  <edge-5p>+</edge-5p>
                  <edge-3p>+</edge-3p>
                  <bond-orientation>c</bond-orientation>
               </base-pair>
               <helix id="H1">
                  <base-id-5p>
                     <base-id>
                        <molecule-id ref="1"/><position>1</position>
                     </base-id>
                  </base-id-5p>
                  <base-id-3p>
                     <base-id>
                        <molecule-id ref="2"/><position>7</position>
                     </base-id>
                  </base-id-3p>
                  <length>7</length>
               </helix>
            </str-annotation>
   </interactions>
</rnaml>
