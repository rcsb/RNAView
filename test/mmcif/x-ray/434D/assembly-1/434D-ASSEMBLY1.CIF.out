PDB data file name: mmcif-test/x-ray/434D/assembly-1/434d-assembly1.cif
-----------------------------------------------------------
CRITERIA USED TO GENERATE BASE-PAIR: 
  3.40 --> upper H-bond length limits (ON..ON).
 26.00 --> max. distance between paired base origins.
  2.50 --> max. vertical distance between paired base origins.
 65.00 --> max. angle between paired bases [0-90].
  5.40 --> min. distance between RN9/YN1 atoms.
  8.00 --> max. distance criterion for helix break[0-12]
-----------------------------------------------------------
BASE-PAIR INSTRUCTIONS: 
Column 1 is rnaview assigned base numbers n1_n2, start from 1.
Column 2 & 3 are chain ID & residue number in input PDB file.
Column 4 is for base pair. The left & right are the bases as 
         identified by column 2 & 3 and 5 & 6.
Column 5 & 6 are residue number & chain ID in input PDB file.
Column 7 is for base pair annotation. The standard Watson-Crick
         (W.C.) pairs are annotated as -/- (AU,AT) or +/+ (GC).
         Other pairs are annotated as Leontis_Westhof Classification.
         The three edges: W(Watson-Crick); H(Hoogsteen); S(suger).
         e.g. W/H means the pair is edge of Watson-Crick & Hoogsteen.
Column 8 is glycosidic bond orientation (either cis or trans).
         e.g. 'W/H cis' means the pair is interaction on Watson-Crick
         and Hoogsteen side, glycosidic bond orientation is 'cis'.
Column 9 corresponds to Saenger Classification.

Other columns: 
        Syn sugar-base conformations are annotated as (syn).
        Stacked base pairs are annotated as (stack).
        Non-identified edges are annotated as (.) or (?)
        Tertiary interactions are marked by (!) in the line.
Reference:
Yang et al (2003) Nucleic Acids Research, Vol31,No13,p3450-3461.
-----------------------------------------------------------
BEGIN_base-pair
     1_14, A:     1 U-A    14 B: -/- cis         XX
     2_13, A:     2 A-U    13 B: -/- cis         XX
     3_12, A:     3 G-C    12 B: +/+ cis         XIX
     4_11, A:     4 C-G    11 B: +/+ cis         XIX
     5_10, A:     5 U-G    10 B: W/W cis         XXVIII
      6_9, A:     6 C-G     9 B: +/+ cis         XIX
      7_8, A:     7 C-G     8 B: +/+ cis         XIX
END_base-pair
  The total base pairs =   7 (from   14 bases)
------------------------------------------------
 Standard  WW--cis  WW-tran  HH--cis  HH-tran  SS--cis  SS-tran
        6        1        0        0        0        0        0
  WH--cis  WH-tran  WS--cis  WS-tran  HS--cis  HS-tran
        0        0        0        0        0        0
------------------------------------------------
